src/main/lua/exasolvs/query_renderer.lua

```diff
@@ -57,15 +57,15 @@ end
 -- @return new query renderer instance
 --
 function M.new (query)
-    local self = {original_query = query, query_elements = {}}
+    local self = { original_query = query, query_elements = {} }
     local OPERATORS = {
         predicate_equal = "=", predicate_notequal = "<>", predicate_less = "<", predicate_greater = ">",
         predicate_and = "AND", predicate_or = "OR", predicate_not = "NOT"
     }
 
     -- forward declarations
     local append_unary_predicate, append_binary_predicate, append_iterated_predicate, append_expression,
-        append_predicate_in, append_select, append_sub_select
+    append_predicate_in, append_select, append_sub_select
```

Comment:
Wrapped lines should get extra indentation. Is that IntelliJ changing the format?